lib/core/form/components/widgets/core/form-field-validator.ts

```diff
@@ -52,7 +52,7 @@ export class RequiredFieldValidator implements FormFieldValidator {
     }
 
     validate(field: FormFieldModel): boolean {
-        if (this.isSupported(field)) {
+        if (this.isSupported(field) && field.isVisible) {
```

Comment:
Could you create a method like isValid that check if the current field is required AND is visible?